docs/get-started/go.md

```diff
@@ -27,15 +27,15 @@ go mod init getuser
 Create a file named **getuser.go** and add the following code.
 
 ```bash
-go get github.com/microsoft/kiota/abstractions/go
```

Comment:
Will need to update links on line 13 as well as they will break after merge.

Reply:
thanks for the callout, fixed.